INSTALL_DOCKER.rst

```diff
@@ -35,15 +32,15 @@ If your user does not already have ``sudo`` rights, you should add your user to
 
   ::
 
-    usermod -G wheel -a <USERNAME>
+    $ usermod -G wheel -a <USERNAME>
 
 This change will take effect once you have logged out as root and logged back in as <USERNAME>.
```

Comment:
We could refer to `newgrp` here too for consistency with the 2nd `usermod` (`$ sudo usermod -G kvm,docker -a <USERNAME>`)?